### ISSUE TITLE
Any goal established for iOS "openDocument" like, on android?

### ISSUE THREAD

**Issue by RUIFERNANDE5** [wkh237/react-native-fetch-blob#357]

**Comment by wkh237** [comment]
@RUIFERNANDE5 , I think [ActionViewIntent](https://github.com/wkh237/react-native-fetch-blob/wiki/Android-API#androidactionviewintentpath-mimepromise) might fulfill your need.

**Comment by RUIFERNANDE5** [comment]
I mean, doesn't the actionViewIntent opens automatically the file based on mime type? I want the share sheet like iOS

**Comment by wkh237** [comment]
@RUIFERNANDE5 , from my understanding, Android does not have a built-in file opener like iOS. After some investigations, I think [this one](https://github.com/iPaulPro/aFileChooser) looks promising. However, I'm wondering if it's a good idea to integrate a large module into this library.

If this is an urgent case to you, I'd suggest making an adapter for `aFileChooser` by following the [guidance in official document](https://facebook.github.io/react-native/docs/native-modules-android.html) 👍

**Comment by RUIFERNANDE5** [comment]
@wkh237 sure, I can try I just don't know if `aFileChooser` is the best library since it has a bunch of issues, pr's and it was not updated for 3+ years. 
I don't really know java, if I can't deal with the creating the adapter but I do find any better library to integrate with `react-native-fetch-blob` i'll make sure to post it there in case someone wants to help! 
Thank you

**Comment by wkh237** [comment]
@RUIFERNANDE5 , please feel free to share your findings. We can still make an experiment in development branch 👍

**Comment by RUIFERNANDE5** [comment]
@wkh237 I was analyzing the library u sent, and I don't know if u really understood what i wanted.
So i'll just post a screenshot so you, or anyone who comes around understands better.

Resumé: I've generated a `.vcf` file and saved it on `DocumentDirectoryPath` folder, now, I need to be able to share this file to any supported app for this mime type (contacts app, email, etc).

![screen shot 2017-05-15 at 14 52 20](https://cloud.githubusercontent.com/assets/7592536/26060861/344fdcd6-397e-11e7-8c5b-f95953d8ee3b.png)
  
Anyway, sorry for the spam and you do not have to answers this question. Just in case you didn't get it or anyone else doesn't.

**Comment by wkh237** [comment]
@RUIFERNANDE5 , have you tried [react-native-share](https://github.com/EstebanFuentealba/react-native-share) ?

**Comment by RUIFERNANDE5** [comment]
@wkh237 yes I have and it is what I'm using for android. Although I'm not happy with the non-native feeling that their share box have. 
How hard would it be to implement the following? I think its the system native share box, at least all system apps do that.

![img_20170515_152232](https://cloud.githubusercontent.com/assets/7592536/26062386/8f8c8b7c-3982-11e7-9c69-c2e4c645be9e.png)


Plus, having all that on just a single library like react-native-fetch-blob would be just great.
